mantidproject/mantid
# Add new Elwin parameter to ISIS instruments

Fixes [#11861](http://trac.mantidproject.org/mantid/ticket/11861).

To test:
- Load a BASIS file from testing instctions in #795 
- See that log value option is `average`
- Load `Babylon5/Public/DanNixon/elwin/irs_red.nxs` (or load `irs26176_graphite002_red.nxs` and run LoadParameterFile on it to update the NeXus instrument)
- Load it in Elwin
- See that the log value option is `last_value`

No release note changes as this was added in #795.
